.travis.yml

```diff
@@ -13,7 +13,12 @@ install:
   - 'gem install compass'
   - 'npm update'
   - 'bower update'
-  
+  - 'node_modules/protractor/bin/webdriver-manager update --standalone --chrome'
```

Comment:
When you get a chance, document these either with `#` comments or in the Dockstore primer